src/main/java/luke/Main.java

```diff
@@ -0,0 +1,149 @@
+package luke;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+import luke.exception.*;
+import luke.task.Deadline;
+import luke.task.Event;
+import luke.task.Task;
+import luke.task.Todo;
+
+
+public class Main extends Application {
+    public static void main(String[] args) {
```

Comment:
This method feels very long. Maybe you can consider refactoring this.

A suggestion would be to create a `Parser` class to process the input, and a `Commands` class to contain the logic for each individual command.

Reply:
Thank you so much! I definitely agree that I really need to reorganize my structure.